python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/auto_materialize_policy_scenarios.py

```diff
@@ -190,18 +190,6 @@ def with_auto_materialize_policy(
                     PartitionKeyRange(start="2013-01-05-00:00", end="2013-01-05-03:00")
                 )
             },
-            ("daily", "2013-01-05"): {
```

Comment:
previously, the fact that we *planned* to materialize the hourly parent asset would cause us to examine the downstream daily partitioned asset. we'd then notice that the daily asset was missing, and see if it'd make sense to materialize it. we would then conclude that the hourly parent was outdated (as we haven't materialized it yet, we're just planning to), and we'd get a ParentOutdatedAutoMaterializeCondition, causing us to not materialize the asset. Then, once the hourly asset actually was materialized, we'd go through this process again, this time realizing that it was ok to materialize the downstream.

This is fairly confusing for the user (in my opinion) as the timing of us discovering that the daily asset is missing is basically random. Now, we'll only discover that the asset is missing once the hourly parent is actually materialized.